### PR TITLE
Fix some 32/64 bit compile warnings

### PR DESCRIPTION
**Pull request by Frassle**
This changes some types to size_t and ptrdiff_t in btyaccpa.ske to prevent __int64 to int cast warnings with msvc.

### CODE REVIEW

File: btyaccpa.ske [hunk 1]
```diff
@@ -194,8 +194,8 @@ static Yshort *yylexemes=0;
 int yyparse(void);
 int yylex(void);
 
-static void YYSCopy(YYSTYPE *to, YYSTYPE *from, int size) {
-  int i;                             
+static void YYSCopy(YYSTYPE *to, YYSTYPE *from, size_t size) {
+  size_t i;
```
Comment (ChrisDodd) [inline btyaccpa.ske]: `size_t` is an unsigned type, so this makes the following loop loop forever...

Comment (Frassle) [reply btyaccpa.ske]: Ah yup silly mistake, interesting that I didn't hit this running a parser built with it. Fixed by using ptrdiff_t here.

**Comment by ChrisDodd** [comment]
with the last change, make test seems to be failing with a bunch of `error: unknown type name ‘ptrdiff_t’`.  I've fixed the travis stuff to actually fail when that happens (still getting the hang of Travis integration).

**Comment by ChrisDodd** [comment]
was just a missing `#include <stddef.h>` in the skeleton.

**Comment by Frassle** [comment]
Ah yup was just about to add that but you beat me to it. Didn't notice the travis failure, make test doesn't work on my windows box at all, and the project I included the generated parser in must already be including stddef so didn't see it there. 
Glad to see all simply fixed up, thanks for your help and getting this merged so quickly :+1:

**Pull request merged**